packages/melos/lib/src/workspace.dart

```diff
@@ -137,17 +170,23 @@ class MelosWorkspace {
   }
 
   /// Builds a dependency graph of dependencies and their dependents in this workspace.
-  Future<Map<String, Set<String>>> getDependencyGraph() async {
+  Future<Map<String, Set<String>>> getDependencyGraph({
+    required String sdkPath,
+  }) async {
     final pubDepsExecArgs = ['--style=list', '--dev'];
     final pubListCommandOutput = await Process.run(
       isFlutterWorkspace
           ? 'flutter'
```

Comment:
We should also use the sdkPath here and and wherever else we invoke `flutter` and `dart` to be consistent.

Reply:
e94de99c841d914fc34b77ff24b056448925d896